carina-core/src/main/resources/config.properties

```diff
@@ -75,6 +75,7 @@ track_known_issues=false
 sender_email={must_override}
 sender_pswd={must_override}
 suite_name=NULL
+test_naming_option=NULL
```

Comment:
to make this feature as much flexible as possible I recommend:
1. rename test_naming_option to test_naming_pattern
2. move this property to "Report configuration" block
3. specify default value as
test_naming_pattern={method_name}